src/PlusServer/Commands/vtkPlusGetUsParameterCommand.cxx

```diff
@@ -300,7 +300,7 @@ vtkPlusUsDevice* vtkPlusGetUsParameterCommand::GetUsDevice()
     // No ultrasound device id is specified, auto-detect the first one and use that
     for (DeviceCollectionConstIterator it = dataCollector->GetDeviceConstIteratorBegin(); it != dataCollector->GetDeviceConstIteratorEnd(); ++it)
     {
-      usDevice = vtkPlusUsDevice::SafeDownCast(*it);
+      usDevice = dynamic_cast<vtkPlusUsDevice*>(*it);
```

Comment:
@Sunderlandkyl and @adamrankin it would be good if you double-checked this change. The old line fails to cast `vtkPlusCapistranoVideoSource` into its parent class, `vtkPlusUsDevice`.

Reply:
I think that the old line failed because this line: https://github.com/PlusToolkit/PlusLib/blob/master/src/PlusDataCollection/Capistrano/vtkPlusCapistranoVideoSource.h#L28

Should be:
```vtkTypeMacro(vtkPlusCapistranoVideoSource, vtkPlusUsDevice);```

Reply:
You are right! I will amend the PR.